exchanges/btcmarkets/btcmarkets_wrapper.go

```diff
@@ -342,35 +350,35 @@ func (b *BTCMarkets) GetExchangeHistory(p currency.Pair, assetType asset.Item) (
 
 // SubmitOrder submits a new order
 func (b *BTCMarkets) SubmitOrder(s *order.Submit) (order.SubmitResponse, error) {
-	var submitOrderResponse order.SubmitResponse
+	var resp order.SubmitResponse
 	if err := s.Validate(); err != nil {
-		return submitOrderResponse, err
+		return resp, err
 	}
 
-	if strings.EqualFold(s.OrderSide.String(), order.Sell.String()) {
+	if s.OrderSide == order.Sell {
 		s.OrderSide = order.Ask
 	}
-	if strings.EqualFold(s.OrderSide.String(), order.Buy.String()) {
+	if s.OrderSide == order.Buy {
 		s.OrderSide = order.Bid
 	}
 
-	response, err := b.NewOrder(s.Pair.Base.Upper().String(),
-		s.Pair.Quote.Upper().String(),
-		s.Price,
+	tempResp, err := b.NewOrder(b.FormatExchangeCurrency(s.Pair, asset.Spot).String(),
 		s.Amount,
+		s.Price,
```

Comment:
It looks like you've got these the other way around!
![image](https://user-images.githubusercontent.com/9261323/69113142-3cf21a80-0ad6-11ea-9284-e3c547305919.png)

Reply:
Wow nice pickup